src/Middleware/SocialAuthMiddleware.php

```diff
@@ -261,16 +258,16 @@ protected function _getProfileEntity(
     /**
      * Get new user entity.
      *
-     * It dispatches a `SocialConnect.newUser` event. A listener must return
+     * It dispatches a `SocialConnect.getUser` event. A listener must return
      * an entity for new user record.
      *
      * @param \Cake\Datasource\EntityInterface $profile Social profile entity.
      *
      * @return \Cake\Datasource\EntityInterface User entity.
      */
-    protected function _newUser(EntityInterface $profile)
+    protected function _getUser(EntityInterface $profile)
```

Comment:
There's already a `_getUser()` method in the class. Could name this `_getUserEntity()` for consistency with `_getProfileEntity()`.